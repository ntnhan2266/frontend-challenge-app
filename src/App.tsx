import React from "react";

import ShapeContainer from "components/containers/ShapeContainer";
import "App.css";

function App() {
  return (
    <div className="App">
      <ShapeContainer />
    </div>
  );
}

export default App;
