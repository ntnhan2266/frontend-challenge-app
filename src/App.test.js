import React from "react";
import { shallow } from "enzyme";
import App from "App";
let wrapped = shallow(<App />);

describe("App", () => {
  it("should render the App Component correctly", () => {
    expect(wrapped).toMatchSnapshot();
  });
});
