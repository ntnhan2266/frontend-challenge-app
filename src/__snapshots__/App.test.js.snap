// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App Component correctly 1`] = `
<div
  className="App"
>
  <ShapeContainer />
</div>
`;
